tests/runtest.proj

```diff
@@ -316,6 +316,9 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
       <AllXUnitFacts Include= "%(AllCommands._XunitFact)" />
     </ItemGroup>
 
+    <Message Text="HasExcludes: $(HaveExcludes)" Importance="high"/>
+    <Message Text="CanonicalExcludeList: @(CanonicalExcludeList)" Importance="high"/>
+
```

Comment:
Looks like debug output that should be removed? If not, HasExcludes -> HaveExcludes.

Reply:
I had previously thought to include this as the output is useful and it is annoying to try to redo the output if needed. Instead I will comment it out so it is possible to uncomment it if further investigation is required.